<!--xml-->
<launch>
    <node name="time_master" pkg="icarus_rover_v2" type="time_master"  output="screen">
        <param name="loop_rate"             value="500"/>
	    <param name="verbosity_level"       value="DEBUG"/>
        <param name="require_pps_to_start"  value="false"/>    
    </node>

    <node name="time_slave" pkg="icarus_rover_v2" type="time_slave"  output="screen">
        <param name="loop_rate"             value="500"/>
	    <param name="verbosity_level"       value="DEBUG"/>
        <param name="require_pps_to_start"  value="true"/>    
    </node>
 
    <node name="diagnostic_node" pkg="icarus_rover_v2" type="diagnostic_node"  output="screen">
        <param name="loop_rate"       value="100"/>
	    <param name="verbosity_level" value="DEBUG"/>
        <param name="require_pps_to_start" value="false" />
        <param name="RAM_usage_threshold_MB"    value="50" />
        <param name="CPU_usage_threshold_percent"    value="5" />
    </node>
</launch>
